yarn.lock

```diff
@@ -9499,7 +9668,7 @@ electron-updater@^4.3.5:
     lodash.isequal "^4.5.0"
     semver "^7.3.2"
 
-electron@*, electron@^11.2.1:
+electron@*:
```

Comment:
`electron@*:` still points to 11.2.1 - is that intended?

Reply:
Good catch. I don't think it should be as nothing depends on it, I'll dig into that one.

Reply:
887d0ae